DEFINITION.md

```diff
@@ -39,3 +39,13 @@ Cloud Native Computing Foundationは、オープンソースでベンダー中
 이 기술은 회복성이 있고, 관리 편의성을 제공하며, 가시성을 갖는 느슨하게 결합된 시스템을 가능하게 한다. 견고한 자동화와 함께 사용하면, 엔지니어는 영향이 큰 변경을 최소한의 노력으로 자주, 예측 가능하게 수행할 수 있다.
 
 Cloud Native Computing Foundation은 벤더 중립적인 오픈소스 프로젝트 생태계를 육성하고 유지함으로써 해당 패러다임 채택을 촉진한다. 우리 재단은 최신 기술 수준의 패턴을 대중화하여 이런 혁신을 누구나 접근 가능하도록 한다.
+
+## Português Brasileiro:
+
+Tecnologias nativas ao cloud empoderam empresas a criarem e rodarem aplicações escaláveis em ambientes modernos e dinâmicos, como núvens públicas, privadas e híbridas. Containeres, service meshes, microservices, infraestruturas imutáveis e APIs declarativas são alguns exemplos dessa estratégia.
```

Comment:
Wdyt about don't translate the `cloud native` term?

Reply:
Maybe translate `microservices` to `microsserviços`

Reply:
I was thinking to keep `cloud native`, but I decided to translate it because:

1. the other translations also translate it

2. if the goal of this definition is to be very easy to understand (yet truthful to the original, of course) to brazilian people, maybe we should keep in english only terms that we also use on a daily basis when talking in portuguese.

Regarding `microservices`: agreed and changed.